src/bindings/wasm/tvgWasmDefaultFont.h

```diff
@@ -97,10 +97,42 @@
 };
 
+inline const char* get_default_font() {
```

Comment:
1. please double check conventions:
` get_default_font()` -> ` getDefaultFont()` 

But there is the only one font, no other fonts existed:
`getDefaultFont()` -> `getFont()`

if  `set/get` is not quite mandatory (we can figure out with the parameters and return types)
`getFont()` -> `font()`

My suggestion:
`font()`

2. Please see other wrong convention naming in the next.